css/style.css

```diff
@@ -43,22 +43,6 @@ form input, form textarea {
 	background-color: #e6e6e6;
 }
 
-table#question-page-table {
```

Comment:
Good that the borders were removed. Looks cleaner and there is no need for any more debugging. Win-Win